README.md

```diff
@@ -67,28 +65,103 @@ or
 ```
 
 ## Usage:
-```
+
+```shell
+❯❯❯ miqsel --help
+
 Usage: miqsel [OPTIONS] COMMAND [ARGS]...
 
+  Miq Selenium Server
+
 Options:
   --version  Show the version and exit.
   --help     Show this message and exit.
 
 Commands:
-  browser   Set Browser
-  config    Configure Miq Selenium webdriver
-  hostname  Miq Selenium Server Hostname
-  start     Start Miq Selenium Server
-  status    Status of Miq Selenium Server
-  stop      Stop Miq Selenium Server
-  urls      VNC and Command Executor URL's
-  viewer    VNC viewer
+  appliance  Appliance
+  browser    Set Browser
+  config     Configure Miq Selenium Server
+  executor   Selenium executor URL
+  start      Start Selenium Server
+  status     Status of Selenium Server
+  stop       Stop Selenium Server
+  viewer     VNC viewer
+  vnc        VNC URL
 ```
-- **Note**: If the project directory `not set` then you need to run commands from `integration_tests` directory. 
 
-* set-project-directory:
+- **Configuration**:
+If the project directory `not set` then `miqsel` need to run from `integration_tests` project directory.  It will help to manage `env.local.yaml`.
+
+    ```shell
+    ❯❯❯ miqsel config
+    Miq project working dir: /home/.../integration_tests
     ```
-    cd `<path>/integration_tests`
-    miqsel config
-    # follow options
+
+- **Status**:
+Check status of selenium container (running/ stopped).
+    ```shell
+    ❯❯❯ miqsel status
+    stopped
+    ```
+
+- **Start/Stop**:
+The start will set command executor in `env.local.yaml` and provide `vnc` access.
+    ```shell
+    ❯❯❯ miqsel start
+    Selenium Server started
+    
+    TigerVNC Viewer 64-bit v1.9.0
+    
+    ❯❯❯ miqsel stop
+    
+    Sun Apr 28 12:29:02 2019
+     CConn:       End of stream
+    Selenium Server stopped
+    ```
+
+- **Appliance**:
+Check current appliance or set new.
+    ```shell
+    ❯❯❯ miqsel appliance
+    
+    ❯❯❯ miqsel appliance -s 192.168.1.1
+    Appliance set to 192.168.1.1
+    ❯❯❯ miqsel appliance
+    192.168.1.1
+    ```
+
+- **Browser**:
+Check current browser or set other.
+    ```shell
+    ❯❯❯ miqsel browser
+    chrome
+    ❯❯❯ miqsel browser --firefox
+    Browser set to firefox
+    ❯❯❯ miqsel browser
+    firefox
+    ```
+
+- **Viewer**:
+Take `vnc` access of selenium server. At start of selenium server automatically `vnc` viewer will 
+`invoke`. If the `status` is `running` and the `viewer` is `close` then viewer help to reopen vnc session.
+
```

Comment:
We can update above line like this
**Optional:**

At the start of selenium server, `vnc` viewer will be `invoked` automatically. If the `status` is `running` and  `viewer` is `closed` then ```viewer``` command will help to re-open vnc session.

Reply:
Added some changes....